packages/datetime/examples/common/formatSelect.tsx

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ * Licensed under the BSD-3 License as modified (the “License”); you may obtain a copy
+ * of the license at https://github.com/palantir/blueprint/blob/master/LICENSE
+ * and https://github.com/palantir/blueprint/blob/master/PATENTS
+ */
+
+import { Radio, RadioGroup } from  "@blueprintjs/core";
+import * as React from "react";
+
+export interface IFormatSelectProps {
+    /**
+     * The format-string option to display as selected.
+     */
+    selectedValue: string;
+
+    /**
+     * The callback to fire when the selected value changes.
+     */
+    onChange: (event: React.FormEvent<HTMLElement>) => void;
+}
+
+export const FormatSelect: React.SFC<IFormatSelectProps> = (props) => (
+    <RadioGroup
+        label="Date format"
+        onChange={props.onChange}
+        selectedValue={props.selectedValue}
+    >
+        <Radio label="DD/MM/YYYY" value="DD/MM/YYYY" />
+        <Radio label="MM-DD-YYYY" value="MM-DD-YYYY" />
+        <Radio label="YYYY-MM-DD" value="YYYY-MM-DD" />
```

Comment:
can you pull these out to an array that is also exported? so we can set `format: FormatSelect.values[0]` in other components' state.